index.html

```diff
@@ -338,7 +338,8 @@ <h3>
         </p>
         <ol class="algorithm">
           <li>Let |document:Document| be the [=environment settings object /
-          responsible document=] of the <a>current settings object</a>.
+          responsible document=] of <a>this</a>' <a>relevant settings
```

Comment:
Nit: here and elsewhere, `this's` instead of `this'`.

Reply:
Done in #309 (I'll rebase this and adjust the remaining occurrences after it lands).